### ISSUE TITLE
SQLite escaping doesn't work

### ISSUE THREAD

**Issue by chanibal** [eveningkid/denodb#149]
Hi, I tried to use denodb for storing html in a SQLite database. And it looks like adding HTML to a `DataTypes.STRING` causes a SQL syntax error. This probably means a SQL-injection is also possible, but did not verify this.

This is the first time I use both deno and denodb, so it might be some kind of misconfiguration. Please correct me if I'm wrong.

```javascript
import { DataTypes, Database, Model } from 'https://deno.land/x/denodb/mod.ts';

const db = new Database('sqlite3', {
  filepath: './data-v2.sqlite',
});


class IssueExample extends Model {
  static table = "IssueExample";
  static fields = { 
    id: { primaryKey: true, autoIncrement: true},
    body: {type: DataTypes.STRING, allowNull: false }
  }
}
db.link([IssueExample]);
await db.sync({ drop: true });

await IssueExample.create({body: "no issue here"});
await IssueExample.create({body: "semicolons ; are an issue"});
```

Exception:
```
error: Uncaught (in promise) SqliteError: unrecognized token: "'semicolons ;"
    return new SqliteError(msg, code);
           ^
    at DB._error (db.ts:289:12)
    at DB.query (db.ts:124:18)
    at sqlite3-connector.ts:55:37
    at Array.map (<anonymous>)
    at SQLite3Connector.query (sqlite3-connector.ts:54:32)
    at async Database.query (database.ts:159:21)
```

**Comment by chanibal** [comment]
Related: https://github.com/eveningkid/denodb/issues/128

**Comment by eveningkid** [comment]
Alright, I know it's certainly too late, but @stillalivx just patched this issue!

It's now available in `1.0.34` so remember to update if you're coming across this issue :)

Have a great day
